project-base/composer.json

```diff
@@ -90,27 +90,30 @@
         "shopsys/product-feed-zbozi": "dev-master",
         "shopsys/product-feed-google": "dev-master",
         "snc/redis-bundle": "2.1.0",
-        "stof/doctrine-extensions-bundle": "1.3.0",
-        "symfony/assetic-bundle": "2.8.2",
-        "symfony/monolog-bundle": "3.1.2",
+        "stof/doctrine-extensions-bundle": "^1.3.0",
+        "symfony/assetic-bundle": "^2.8.2",
+        "symfony/monolog-bundle": "^3.1.2",
         "symfony/swiftmailer-bundle": "^3.2.2",
         "symfony/symfony": "^3.4.8",
-        "symfony-cmf/routing": "2.0.3",
-        "symfony-cmf/routing-bundle": "2.0.3",
-        "timwhitlock/jparser": "dev-get-rid-of-import#c37f4c6",
-        "tracy/tracy": "2.4.13",
-        "twig/extensions": "1.5.1",
-        "twig/twig": "1.35.3",
-        "vasek-purchart/console-errors-bundle": "1.0.1"
+        "symfony-cmf/routing": "^2.0.3",
+        "symfony-cmf/routing-bundle": "^2.0.3",
+        "timwhitlock/jparser": "@dev",
```

Comment:
Please, consider original solution - using specific version dev-get-rid-of-import#c37f4c6.